src/explorer.api/Models/DiffixApiClient.cs

```diff
@@ -226,10 +257,23 @@ async public Task<QueryResult<RowType>> PollQueryResult<RowType>(string queryId)
                 );
         }
 
+        /// <summary>
+        /// Calls <code>PollQueryResult</code> in a loop until the query completes. Can be canceled.
+        /// </summary>
+        /// <remarks>
+        /// Canceling via the <code>CancellationToken</code> cancels the returned <code>Task</code> but does not
+        /// cancel query execution. To do this, a call to <code>/api/queries/{query_id}/cancel</code> must be made.  
+        /// </remarks>
+        /// <param name="queryId">The query Id obtained via a previous call to the /api/query endpoint.</param>
+        /// <param name="ct">A <code>CancellationToken</code> that cancels the returned <code>Task</code>.</param>
+        /// <param name="pollFrequency">How often to poll the api endpoint. Default is 500 milliseconds</param>
```

Comment:
What is the unit?

Reply:
oh never mind, I guess a TimeSpan contains it's own unit..?

Reply:
Yes exactly, you create a `Timespan` like: `TimeSpan.FromSeconds(5)`.